rastervision/command/chip_command_config.py

```diff
@@ -68,7 +69,8 @@ def split(self, num_parts):
         commands = []
         t_scenes = list(map(lambda x: (0, x), self.train_scenes))
         v_scenes = list(map(lambda x: (1, x), self.val_scenes))
-        group_size = max(int((len(t_scenes) + len(v_scenes)) / num_parts), 1)
+        group_size = max(
```

Comment:
It seems like the logic for getting group size should be moved into a utility function since it is duplicated in `predict_command_config`. You may also want to consider combining that and the `grouped` function into a single `split_into_groups(list, num_groups)` since they usually co-occur. In addition, I think this function should be unit tested so we can make sure it is correct in edge cases, like more groups than elements in list, number of elements isn't divisible by number groups, etc.